api/toc.yml

```diff
@@ -1,10 +1,12 @@
-- name: Q# library reference
-  href: qsharp/toc.yml
-- name: C# libraries
+- name: Reference
   items:
-  - name: Trace simulator 
-    href: https://docs.microsoft.com/dotnet/api/microsoft.quantum.simulation.simulators.qctracesimulators
-  - name: Quantum chemistry
-    href: https://docs.microsoft.com/dotnet/api/microsoft.quantum.chemistry
-- name: Python libraries
-  href: https://docs.microsoft.com/python/qsharp    
\ No newline at end of file
+  - name: Q# libraries
+    href: qsharp/toc.yml
+  - name: C# libraries
```

Comment:
Do we want to take the opportunity to emphasize that these are .NET libraries, not just C#?

Reply:
Yep, can do.